platform/android/app/src/main/kotlin/coop/polypoly/polypod/homescreen/HomeScreenViewModel.kt

```diff
@@ -21,12 +25,12 @@ enum class SectionType {
     DEVELOPER;
 
     companion object {
-        fun fromCategoryType(type: FeatureCategory): SectionType {
+        fun fromCategoryType(type: FeatureCategoryId): SectionType {
             return when (type) {
-                FeatureCategory.yourData -> YOUR_DATA
-                FeatureCategory.knowHow -> DATA_KNOW_HOW
-                FeatureCategory.tools -> TOOLS
-                FeatureCategory.developer -> DEVELOPER
+                FeatureCategoryId.yourData -> YOUR_DATA
```

Comment:
Also, maybe just create a constant map? This is just mapping type to constants; it can just be a constant map. There's no logic involved.

Reply:
I will remove this mapping, but in general I would prefer a switch statement instead of constant map - **the switch statement will throw error if not all variants are mapped while the constant map will be silent**.